ipatests/test_integration/test_automount_locations.py

```diff
@@ -0,0 +1,75 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+
+"""
+This module provides tests for the automount location feature.
+"""
+
+from __future__ import absolute_import
+
+import time
+
+from ipatests.test_integration.base import IntegrationTest
+from ipatests.pytest_ipa.integration import tasks
+
+# give some time for units to stabilize
+# otherwise we get transient errors
+WAIT_AFTER_INSTALL = 5
+WAIT_AFTER_UNINSTALL = WAIT_AFTER_INSTALL
+
+
+class TestAutomountInstallUninstall(IntegrationTest):
+    """
+    Test if vault feature behaves as expected, when KRA is installed or not
+    installed on replica
+    """
+
+    num_replicas = 1
```

Comment:
I think you can use ``num_replicas = 0`` and ``num_clients = 1`` here, too. That would allow you to avoid one expensive ipa-replica-install and just use a faster client installation. Of course you need to change topology to ``topology: *master_1repl_1client``.

Perhaps it's time to introduce a ``topology: *master_1client``. I have several other tests that just need a master and one client, too.

Reply:
We will need two clients in an upcoming version of that test. I'll see if there a topology that fits better.

Reply:
Actually what I would like for the next class (which will not be part of the current PR but comes in next sprint) is master_2clients but we don't seem to have that. Please let me know if we can resolve this now.